runtimes/php-8.0/build.sh

```diff
@@ -1 +1,7 @@
-composer install --no-interaction --no-dev --prefer-dist --optimize-autoloader
\ No newline at end of file
+cd /usr/local/src/
+composer update --no-interaction --ignore-platform-reqs --optimize-autoloader --no-scripts --prefer-dist --no-dev
+
+cd /usr/code/
+if [[ -f "composer.json" ]]; then
```

Comment:
I like the way that this was executed. It is smart to include this check on the build script.